### DISCUSSION TITLE
executing Cmd() from LUA without the confirmation in command line history

### DISCUSSION THREAD

**Discussion by lukas-runge** [hossimo/GMA3Plugins#27] [Q&A]
Hey everyone,

is there a way to silently excecute stuff with LUA so the user does not notice something has been executed even when he takes a look at command line history?

Kind regards
@lukas-runge

**Comment by hossimo** [comment]
Using Cmd() I doubt it. However you can do many if not all things directly in Lua, often times you can do this via the [ShowData Data Object](https://github.com/hossimo/GMA3Plugins/wiki/ShowDataDataObject) or by looking at the included hardware and system test plugins.

Unfortunately I have not documented all of the DataObjects (was very time intensive and error-prone) and apparently I have not linked them to the main article above, but you can search or the ones included from the pages _find a page_ search box by looking for "**DataObject**", hopefully you find one useful.
